frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -1447,550 +1874,150 @@ function ReaderFooter:addToMainMenu(menu_items)
-                        title_text = _("Minimal width"),
-                        text = _("Minimal progress bar width in percentage of screen width"),
-                        keep_shown_on_apply = true,
-                        callback = function(spin)
-                            self.settings.progress_bar_min_width_pct = spin.value
-                            self:refreshFooter(true, true)
-                            if touchmenu_instance then touchmenu_instance:updateItems() end
-                        end
-                    }
-                    UIManager:show(items)
-                end,
-                keep_menu_open = true,
-            }
         }
     })
-    table.insert(sub_items, getMinibarOption("page_progress"))
-    table.insert(sub_items, getMinibarOption("pages_left_book"))
-    table.insert(sub_items, getMinibarOption("time"))
-    table.insert(sub_items, getMinibarOption("chapter_progress"))
-    table.insert(sub_items, getMinibarOption("pages_left"))
+    local configure_complications_sub_table = sub_items[#sub_items].sub_item_table -- will pick the last item of sub_items
     if Device:hasBattery() then
-        table.insert(sub_items, getMinibarOption("battery"))
-    end
-    table.insert(sub_items, getMinibarOption("bookmark_count"))
-    table.insert(sub_items, getMinibarOption("percentage"))
-    table.insert(sub_items, getMinibarOption("book_time_to_read"))
-    table.insert(sub_items, getMinibarOption("chapter_time_to_read"))
-    if Device:hasFrontlight() then
-        table.insert(sub_items, getMinibarOption("frontlight"))
+        table.insert(configure_complications_sub_table , 5, {
+            text_func = function()
+                if self.settings.battery_hide_threshold <= (Device:hasAuxBattery() and 200 or 100) then
+                    return T(_("Hide battery item when higher than: %1".. "%"), self.settings.battery_hide_threshold)
+                else
+                    return _("Hide battery item at custom threshold")
+                end
+            end,
+            checked_func = function()
+                return self.settings.battery_hide_threshold < MAX_BATTERY_HIDE_THRESHOLD
+            end,
+            enabled_func = function()
+                return self.settings.all_at_once == true
+            end,
+            separator = true,
+            callback = function(touchmenu_instance)
+                local SpinWidget = require("ui/widget/spinwidget")
+                local battery_threshold = SpinWidget:new{
+                    value = math.min(self.settings.battery_hide_threshold, Device:hasAuxBattery() and 200 or 100),
+                    value_min = 0,
+                    value_max = Device:hasAuxBattery() and 200 or 100,
+                    default_value = Device:hasAuxBattery() and 200 or 100,
+                    unit = "%",
+                    value_hold_step = 10,
+                    title_text = _("Set minimum threshold to hide battery item"),
```

Comment:
Feels a bit long and ugly with the resulting small font size:
![image](https://github.com/koreader/koreader/assets/24273478/2b6fa978-6939-41cf-bb75-285904f6e2f5)
In other places, we get:
![image](https://github.com/koreader/koreader/assets/24273478/f6999ad5-619d-4ee1-a96e-f1ebcaaf8da2)

(Remove "Set", use "min", or reword that?)